packages/table/_table-head.scss

```
@@ -19,6 +21,11 @@
             top: 0;
             z-index: 1;
             background-color: var(--jkl-table-head-sticky-color);
+
+            border-bottom: none;
+
+            box-shadow: inset 0 0 0 #000, inset 0 $_border-size 0 var(--jkl-table-row-border-color);
```

```suggestion
            box-shadow: inset 0 0 0 #000, inset 0 $_border-size 0 var(--jkl-table-row-hover-border-color);
```

Borderen under header skal være granitt/snøhvit.

Denne skyggen rendrer for øvrig ikke i Safari 16.1 med mindre den settes på `th` i stedet.

---

Oh takk! <3

Hmm, problemet med å sette den på `th` er kolonner som med `sr-only`